cards/common-partials/thumbsfeedback.hbs

```diff
@@ -0,0 +1,43 @@
+{{#if card.feedback}}
```

Comment:
Could this partial be used with the DA cards as well? It seems like it would be nice for both results and DA cards to use the same partial for feedback.